src/app/app.module.ts

```diff
@@ -10,16 +10,17 @@ import { ConfigService } from './config.service';
 import {AuthModule} from './auth/auth.module';
 import {HmctsModule} from './hmcts/hmcts.module';
 import {GovukModule} from './govuk/govuk.module';
+import {RouterModule} from '@angular/router';
 
 @NgModule({
     declarations: [
         AppComponent
     ],
     imports: [
+        RouterModule,
```

Comment:
Why did u decided to remove routes from routerModule?

Reply:
I have added it not removed it

Reply:
this was removed ` // RouterModule.forRoot(routes),` so that what caught my attention